### PR TITLE
NO-ISSUE: print boot order after setting it

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

**Comment by omertuc** [comment]
Why the go.mod change?

File: src/ops/ops.go [hunk 1]
```diff
@@ -192,6 +192,10 @@ func (o *ops) SetBootOrder(device string) error {
 		return err
 	}
 	o.handleDuplicateEntries(out)
+	out, err = o.ExecPrivilegeCommand(o.logWriter, "efibootmgr", "-l", o.getEfiFilePath())
```
Comment (eranco74) [inline src/ops/ops.go:195]: you don't use `out`

**Review by eranco74 — COMMENTED** [review]
Just a nit

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Comment by tsorya** [comment]
changed automatically

**Pull request merged**